src/benchmarklib/jcch/jcch_benchmark_item_runner.cpp

```diff
@@ -36,7 +36,7 @@ JCCHBenchmarkItemRunner::JCCHBenchmarkItemRunner(const bool skewed, const std::s
 }
 
 std::string JCCHBenchmarkItemRunner::item_name(const BenchmarkItemID item_id) const {
-  Assert(item_id < 22u, "item_id out of range");
+  Assert(item_id < 22u, "item_id out of range.");
```

Comment:
I don't think we have a guideline for that. For me, just using `u` is either not specified properly or unnecessary.
In case we use `auto a = 10u;` it contradicts our statement on being explicit about types (e.g., `uint{32,64}_t`).
In this case here, we have the type BenchmarkItemID, which is of type `size_t`. Thus, to it's like saying `auto item_id = size_t{10u};` where the `u` just isn't needed.

Reply:
I'm on the same track here, just wanted to get your opinion and remove it as well. Do we want to state that somewhere in the giudelines?

Reply:
Soo changes to `CONTRIBUTING.md`.